keadm/cmd/keadm/app/cmd/reset.go

```diff
@@ -103,6 +118,36 @@ func TearDownKubeEdge(isEdgeNode bool, kubeConfig string) error {
 	return nil
 }
 
+// RemoveContainers removes all Kubernetes-managed containers
+func RemoveContainers(isEdgeNode bool, execer utilsexec.Interface) error {
+	if !isEdgeNode {
+		return nil
+	}
+
+	criSocketPath, _ := utilruntime.DetectCRISocket()
+
+	containerRuntime, _ := utilruntime.NewContainerRuntime(execer, criSocketPath)
+
+	containers, _ := containerRuntime.ListKubeContainers()
+
+	return containerRuntime.RemoveContainers(containers)
+}
+
+func cleanDirectories(isEdgeNode bool) error {
+	var dirToClean = []string{"/var/lib/edged", "/etc/kubeedge"}
+
+	if isEdgeNode {
+		dirToClean = append(dirToClean, "/var/lib/dockershim", "/var/run/kubernetes", "/var/lib/cni")
+	}
+
+	// TODO: more mount directories?
+
+	for _, dir := range dirToClean {
+		_ = phases.CleanDir(dir)
+	}
+	return nil
```

Comment:
Please handle all errors in `RemoveContainers` and `cleanDirectories`

Reply:
Added.